src/olympia/users/tests/test_views.py

```diff
@@ -442,8 +442,7 @@ def test_success(self):
         assert self.user.email == 'jbalogh@mozilla.com'
         url = reverse('users.emailchange', args=[self.user.id, self.token,
                                                  self.hash])
-        r = self.client.get(url, follow=True)
-        assert r.status_code == 200
```

Comment:
It was a `200` before because it redirected to the login page. Now that the login page is on FxA you get a 404 because it doesn't end up on a valid page.